open-metadata-implementation/access-services/asset-catalog/asset-catalog-server/src/main/java/org/odpi/openmetadata/accessservice/assetcatalog/auditlog/AssetCatalogAuditCode.java

```diff
@@ -43,10 +45,28 @@
             "The local server has requested shut down of the Asset Catalog OMAS.",
             "No action is required.  This is part of the normal operation of the server."),;
 
+    /**
+     * the unique identifier for the error message.
+     */
     private String logMessageId;
```

Comment:
This style will most likely need to be turned back into explicit methods when we add localization support.